checkout/access-checkout-android/src/main/java/com/worldpay/access/checkout/session/request/handlers/VerifiedTokensSessionRequestHandler.kt

```diff
@@ -17,17 +17,34 @@ internal class VerifiedTokensSessionRequestHandler(
     private val sessionRequestHandlerConfig: SessionRequestHandlerConfig
 ) : SessionRequestHandler {
 
+    /**
+     * Accepts a list of [SessionType]s and returns True if the list contains a [VERIFIED_TOKEN_SESSION]
+     */
     override fun canHandle(sessionTypes: List<SessionType>): Boolean {
         return sessionTypes.contains(VERIFIED_TOKEN_SESSION)
     }
 
+    /**
+     * Accepts in a [CardDetails] object and validates that the mandatory fields for this [SessionType] are present
+     *
+     * Mandatory fields:
+     * @param cardDetails.pan
+     * @param cardDetails.expiryDate
+     * @param cardDetails.cvv
+     */
     override fun handle(cardDetails: CardDetails) {
         validateNotNull(cardDetails.pan, "pan")
         validateNotNull(cardDetails.expiryDate, "expiry date")
         validateNotNull(cardDetails.cvv, "cvv")
 
+        /**
+         * Retrieves external [SessionResponseListener] and notifies that the request has started
```

Comment:
We don't need comments in here as the line is quite easy to understand by just reading it, also within functions, use in-line comments instead of block comments